README.md

```diff
@@ -184,124 +157,52 @@ There is an enum to use these modes.
+Returns the build authorization header in the following format when using `FSXARemoteApi`:
 
 ```typescript
 {
   authorization: 'apikey="xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"'
 }
 ```
 
-Returns an empty object when mode is set to `proxy`.
+Returns an empty object when using `FSXAProxyApi`.
 
 Example:
 
 ```typescript
-fsxaApi.buildAuthorizationHeaders()
+const remoteApi = new FSXARemoteApi(config)
+const authorizationHeader = remoteApi.authorizationHeader
+// console.log { authorization: `apikey="${config.apikey}"` }
 ```
 
 ### buildCaaSURL
 
-Expects an optional parameter `remoteProjectId`. If passed, the CaaS Url will be built with it instead of the default project id.
-
-Returns the build CaaS url when mode is set to `remote`:
-
-Returns an empty string when mode is set to `proxy`.
+This methods builds an URL for the CaaS. Based upon the optional `buildCaaSUrlParams` object the returning url can link to any desired document.
 
 Example:
 
 ```typescript
-fsxaApi.buildCaaSUrl()
+remoteApi.buildCaaSUrl({ id, locale, remoteProject, additionalParams, filters, page, pagesize, sort, }?: buildCaaSUrlParams): string;
 ```
 
 ### buildNavigationServiceUrl
 
-Returns the build navigation-service url when mode is set to `remote`:
-
-Returns an empty string when mode is set to `proxy`.
+This function builds the URL for the NavigationService. Based upon the optional `buildNavigationServiceURLParams` object the returning url can link to a seo route or a locale specific subtree of the navigation. For more details how the Navigation Service works, read the Navigation
 
 Example:
 
 ```typescript
-fsxaApi.buildNavigationServiceUrl()
+remoteApi.buildNavigationServiceUrl({ locale, initialPath, all }?: buildNavigationServiceURLParams): string;
 ```
 
 ### fetchElement
 
-Returns the corresponding CaaS data entry.
-
-Expects as input parameter an id, which is described in CaaS as 'identifier' and a language abbreviation.
-<br />
-Optionally, additional parameters can be passed that will be appended to the CaaS-Url. Be aware that the response is not mapped if you pass the keys. For more information please refer to the [restheart documentation](https://restheart.org/docs/read-docs/#projection).
-<br />
-Optionally, a remoteProject can be passed. If one is passed, the element will be fetched from this project instead of the default one.
-
-In this example the additional parameters ensure that only the fields `identifier` and `displayName` are in the result set:
+This method fetches an element from the configured CaaS. The `FetchElementParams` object defines options to specify your request. Check `buildCaaSUrl` to know which URL will be used.
 
 ```typescript
-fsxaApi.fetchElement('xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', 'en-EN', {
-  keys: [{ identifier: 1 }, { displayName: 1 }],
-})
+remoteApi.fetchNavigation({ locale, initialPath, fetchOptions, filterContext, }: FetchNavigationParams): Promise<NavigationData | null>;
```

Comment:
this looks like a function definition. I think we should stick with working code examples.